ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -1,4 +1,6 @@
 let prototyped_of_class = function
+  | "Tracing" ->
+      Some "23.9.0"
```

Comment:
This version has been released, this file needs to be checked out and regenerated

Reply:
How would I do this?

Reply:
This file is regenerated whenever `make` is run